src/main/scala/pl/project13/kaze/KazeClass.scala

```diff
@@ -467,6 +485,9 @@ object KazeClass {
     }
 
     private def up(s: String): String = s.charAt(0).toUpper + s.tail
+
+    private def hyphenize(s: String): String =
+      s.charAt(0).toLower + s.tail.replaceAll("""([A-Z])""", "\\-$1").toLowerCase(Locale.ENGLISH)
```

Comment:
In Akka we use `s.toLowerCase(Locale.ROOT)`

Reply:
Ping?

Reply:
This code is not part of the generated stuff, changed it anyhow.